src/state/sagas/datasets/CreateDataset/CreateDataset.js

```diff
@@ -23,13 +46,23 @@ export function* createDataset(action) {
       selectedDatasetId: data.id,
     });
 
-    if (dataset.sourceType !== 'None') {
+    if (!['None', 'File'].includes(dataset.sourceType)) {
       yield put({
         type: DATASET_ACTIONS_KEY.TRIGGER_SAGA_REFRESH_DATASET,
         organizationId,
         datasetId: data.id,
       });
     }
+    if (dataset.sourceType === 'File') {
```

Comment:
Really small improvement, but this could be an `else if` to prevent checking the second condition (line 56) when the first one (line 49) is true :slightly_smiling_face: 

Another way to organize the conditions would be:
```js
if (dataset.sourceType === 'File') {
  // handle File case
} else if (dataset.sourceType !== 'None') {
  // handle ADT and Storage cases
} 
```
but this second version would probably break if we add new source types in the future :sweat_smile:

Reply:
I'll take the first option, just in case :laughing: